addons/l10n_ro_efactura/wizard/account_move_send.py

```diff
@@ -0,0 +1,97 @@
+import requests
+
+from odoo import api, fields, models, _
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    l10n_ro_edi_send_enable = fields.Boolean(compute='_compute_l10n_ro_edi_send_enable')
+    l10n_ro_edi_send_readonly = fields.Boolean(compute='_compute_l10n_ro_edi_send_readonly')
+    l10n_ro_edi_send_checkbox = fields.Boolean(
+        string='Send E-Factura to SPV',
+        compute='_compute_l10n_ro_edi_send_checkbox', store=True, readonly=False,
+        help='Send the CIUS-RO XML to the Romanian Government via the ANAF platform')
+    l10n_ro_edi_warnings = fields.Char(compute='_compute_l10n_ro_edi_warnings')  # To be removed in master
+
+    def _get_wizard_values(self):
+        # EXTENDS 'account'
+        values = super()._get_wizard_values()
+        values['l10n_ro_edi_send'] = self.l10n_ro_edi_send_checkbox
+        print(self.l10n_ro_edi_send_enable, self.l10n_ro_edi_send_readonly, self.l10n_ro_edi_send_checkbox)
+        return values
+
+    @api.depends('move_ids.l10n_ro_edi_state', 'enable_ubl_cii_xml')
+    def _compute_l10n_ro_edi_send_enable(self):
+        """ Enable send to SPV if we can create the XML, or
+            if the XML is already created and the move already have a l10n_ro_edi.document in error """
+        for wizard in self:
+            wizard.l10n_ro_edi_send_enable = any(
+                (move._need_ubl_cii_xml() or move.ubl_cii_xml_id) and
+                move.country_code == 'RO' and
+                move.l10n_ro_edi_state in (False, 'invoice_sending')
+                for move in wizard.move_ids
+            )
+
+    @api.depends('move_ids.l10n_ro_edi_state', 'l10n_ro_edi_send_enable')
+    def _compute_l10n_ro_edi_send_readonly(self):
+        """ We shouldn't allow the user to send a new request if any move is currently waiting for an answer. """
+        for wizard in self:
+            wizard.l10n_ro_edi_send_readonly = not wizard.l10n_ro_edi_send_enable or any(
+                move.l10n_ro_edi_state == 'invoice_sending' for move in wizard.move_ids)
```

Comment:
(cosmetic suggestion, ignore if you disagree)
```suggestion
            wizard.l10n_ro_edi_send_readonly = (
                not wizard.l10n_ro_edi_send_enable
                or 'invoice_sending' in wizard.move_ids.mapped('l10n_ro_edi_state')
            )
```